collect_app/src/main/java/org/odk/collect/android/activities/InstanceChooserList.java

```diff
@@ -196,10 +196,10 @@ public void syncComplete(@NonNull String result) {
 
     private void setupAdapter() {
         String[] data = new String[]{
-                InstanceColumns.DISPLAY_NAME, InstanceColumns.DISPLAY_SUBTEXT, InstanceColumns.DELETED_DATE
+                InstanceColumns.DISPLAY_NAME, InstanceColumns.DELETED_DATE
```

Comment:
Maybe the subtext was saved in the DB so that it could be passed into the adapter this way? I know it doesn't particularly matter and maybe it was just an accident but I keep wondering why in the world the string was stored in the DB if there was also a coded representation of the state this whole time!

Reply:
Maybe... maybe the author didn't know it's possible to handle dates and create an appropriate subtext in the adapter.